lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/f2f-pending.yaml

```diff
@@ -16,7 +16,7 @@ PENDING_PAGE:
     pageId: page-ipv-pending
   events:
     next:
-      targetState: RETURN_TO_RP
+      targetState: RESET_SESSION_IDENTITY_BEFORE_RETURN_TO_RP
```

Comment:
This isn't the F2F handoff page - this is the page you see if you return back to IPV Core before going to the post office (and I don't think we'll have loaded anything into the session for that).

I think you're looking for `F2F_HANDOFF_PAGE` instead 🙂

Reply:
ah ok thanks @Joe-Edwards-GDS

Reply:
@Joe-Edwards-GDS please see latest commit :)